packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/flow-diagram.md

```diff
@@ -0,0 +1,34 @@
+### Authorization code
+
+![Authorization Code](/img/authorization/oauth-auth-code-grant-flow.png "Sequence diagram that displays the back and forth between the resource owner, authorization server, and resource server for Auth Code flow")
+
+<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
+
+skinparam monochrome true
+actor "Resource Owner (User)" as user
+participant "Web App" as client
+participant "Authorization Server (Okta) " as okta
+participant "Resource Server (Your App) " as app
+
+autonumber "<b>#."
+client -> okta: Authorization Code Request to /authorize
+okta -> user: 302 redirect to authentication prompt
+user -> okta: Authentication & consent
+okta -> client: Authorization Code Response
+client -> okta: Send authorization code + client secret to /token
+okta -> client: Access token (and optionally Refresh Token)
+client -> app: Request with access token
+app -> client: Response
+-->
+
+At a high-level, this flow has the following steps:
+
+1. Your application (app) directs the browser to the Okta sign-in page.<br>
```

Comment:
do you need the <br> here? Two returns and a tab puts that content on a new line (with an empty line above) and indented under step 1. Does that not work?